lib/util/logger.js

```diff
@@ -196,6 +211,10 @@ Logger.prototype._read = function _read() {
  * already converted the objects to strings.
  * Returns a boolean representing the status of the write
  * (success/failure)
+ *
+ * @param level
+ * @param args
+ * @param extra
```

Comment:
Looks like my editor decided to add some annotations here.  I can remove them if needed.